src/java_tools/import_deps_checker/javatests/com/google/devtools/build/importdeps/golden_test.sh

```diff
@@ -54,7 +54,8 @@ if [[ "${checker_ret}" != ${expected_exit_code} ]]; then
   exit 1 # Exit with an error.
 fi
 
-diff "${gold_output_file}" <($PRINT_JDEPS_PROTO "${output_file}")
+# Correct for different paths between Blaze and Bazel.
+diff <(sed 's|third_party/bazel/||g' "${gold_output_file}") <($PRINT_JDEPS_PROTO "${output_file}")
```

Comment:
This causes test failures when it's used for the internal Blaze tests, because the actual output still contains `third_party/bazel/`, and that has been normalized away in the golden outputs.

One option would be do the normalize to both the actual and expected outputs, here and below.

Reply:
The tenative plan is to just run both the actual and expected files through `sed`. That seems fine to me, but suggestions welcome if anyone has cleaner ideas:

```diff
-diff <(sed 's|third_party/bazel/||g' "${gold_output_file}") <($PRINT_JDEPS_PROTO "${output_file}")
+diff <(sed 's|third_party/bazel/||g' "${gold_output_file}") <($PRINT_JDEPS_PROTO "${output_file}" | sed 's|third_party/bazel/||g')
```

```diff
-diff <(sed 's|third_party/bazel/||g' "${gold_stderr_file}") "${checker_stderr}"
+diff <(sed 's|third_party/bazel/||g' "${gold_stderr_file}") <(sed 's|third_party/bazel/||g' "${checker_stderr}")
```